packages/web-components/src/components/Grid/Grid.tsx

```diff
@@ -0,0 +1,68 @@
+import { clsx } from "clsx";
+import type { ComponentChildren, FunctionComponent } from "preact";
+import type { HTMLAttributes } from "preact/compat";
+import { useMemo, useRef } from "preact/hooks";
+import type { Options } from "sortablejs";
+import styles from "./Grid.module.scss";
+import { useSortable } from "./hooks";
+
+export type GridKind = "default" | "swap";
+
+export interface GridProps extends HTMLAttributes<HTMLDivElement> {
+  /**
+   * Child elements to render within the layout.
+   */
+  children: ComponentChildren;
```

Comment:
As I wrote above – `FunctionalComponent` already has this, just remove the property